app/connectors/addressLookup/AddressLookupConfiguration.scala

```diff
@@ -65,4 +65,45 @@ class AddressLookupConfiguration @Inject()(callbackUrl: String,
     )
   }
 
+  def applyEEA: AddressLookupEEAConfigurationModel = {
+    val english = Lang("en")
+    val welsh = Lang("cy")
+
+    AddressLookupEEAConfigurationModel(
```

Comment:
Can we use AddressLookupOptionsModel model with last parameter as
allowedCountryCodes: Option[Set[String]] = None